data/test_evaluations/end_to_end_trips_circuit_breaker.md

```diff
@@ -0,0 +1,21 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [story_trips_circuit_breaker](#story_trips_circuit_breaker)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
```

Comment:
Looks like `doctoc` added some table of contents to your stories 😄  We should remove them, and from the other file as well. Maybe they got added by the pre-commit hook?

Reply:
Oh boy! Yeah,  I did get some odd warnings about  the pre-commit hook